src/egon/data/processing/openstreetmap.py

```diff
@@ -0,0 +1,62 @@
+from egon.data import db
+import egon.data.config
+
+
+def modify_tables():
```

Comment:
This function should probably be split up into smaller functions. The fact that there are sections with explanatory comments are a good indicator that this should actually be dedicated functions with the comments as their docstrings.

Reply:
I would do that on-demand. If you prefer to file an issue for that already, feel free. Otherwise, please click "resolve".

Reply:
It's related to the next conversation anyway, so I submitted it as #55.